unix/Makefile

```diff
@@ -153,3 +160,16 @@ coverage_test: coverage
 	cd ../tests && MICROPY_MICROPYTHON=../$(DIRNAME)/micropython_coverage ./run-tests --emit native
 	gcov -o build-coverage/py ../py/*.c
 	gcov -o build-coverage/extmod ../extmod/*.c
+
+_frozen_upip.c: frozen_upip/upip.py
```

Comment:
Can _frozen_upip.c go in the $(BUILD)/ directory?  Would be cleaner that way (since it's a generated file).